redis/src/main/scala/zio/redis/Output.scala

```
@@ -706,4 +716,169 @@ object Output {
       case _: NumberFormatException => throw ProtocolError(s"'$text' isn't a double.")
     }
   }
+
+  case object ClientInfoOutput extends Output[Chunk[ClientInfo]] {
+    private def parseLong(s: String): Option[Long] =
+      try Some(s.toLong)
+      catch { case _: NumberFormatException => None }
+
+    private def parseInt(s: String): Option[Int] =
+      try Some(s.toInt)
+      catch { case _: NumberFormatException => None }
```

It looks like these will silently discard errors. Is that the desired behavior?